### PR TITLE
[layout] make client request server layout

### PR DESCRIPTION
**Pull request by JanKoehnlein**
Fixes the example

### CODE REVIEW

File: examples/circlegraph/src/main/ts/main.ts [hunk 1]
```diff
@@ -34,7 +34,9 @@ server.listen(websocket);
 
 const dispatcher = container.get<IActionDispatcher>(TYPES.IActionDispatcher);
 websocket.addEventListener('open', async () => {
-    const setModel = await dispatcher.request(RequestModelAction.create());
+    const setModel = await dispatcher.request(RequestModelAction.create({
+        needsServerLayout: true
```
Comment (spoenemann) [inline examples/circlegraph/src/main/ts/main.ts]: Can't we set this in the ViewerOptions instead?

Comment (JanKoehnlein) [reply examples/circlegraph/src/main/ts/main.ts]: And then pass the viewerOptions here? 
In this case, I'd prefer to have `ViewerOptions` explicitly as an argument of the `RequestModelAction.create` and the constructor.

Comment (spoenemann) [reply examples/circlegraph/src/main/ts/main.ts]: Didn't you already automate that?

https://github.com/eclipse/sprotty/blob/3f7128d154aea538386bcaffb5acc5affc2353b6/src/model-source/diagram-server.ts#L166-L171

Comment (JanKoehnlein) [reply examples/circlegraph/src/main/ts/main.ts]: You are right. Changed it.

Comment (JanKoehnlein) [reply examples/circlegraph/src/main/ts/main.ts]: Had numerous problems with stale build caches, so I must have overseen this.

**Pull request merged**